_posts/2021-06-17-sandstorm-community-fundraising.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Fundraising for the Sandstorm Community"
+author: Ian Denhardt
+authorUrl: https://github.com/zenhack
+---
+
+Hey Everyone!
+
+Members of the Sandstorm community are doing some fundraising!
+
+Specifically,
+
+- In the immediate term, we're [participating in FundOSS][1], a donation
+  matching program that is a joint effort between [OpenCollective[2] and
```

Comment:
Looks like you are missing a ]

Reply:
That was fast! I noticed this basically right after pushing and then force-pushed -- but you were already reviewing!